schemas/JSON/manifests/v1.1.0/manifest.installer.1.1.0.json

```diff
@@ -132,20 +132,56 @@
         }
       }
     },
+    "InstallerReturnCode": {
+      "type": "integer",
+      "not": {
+        "enum": [ 0 ]
+      },
+      "minimum": -2147483648,
+      "maximum": 429496729,

```

Comment:
>429496729 [](http://example.com/codeflow?start=17&length=9)

Did you accidentally change the value, the previous 4294967295 is the max for uint32

Reply:
Yes, it was accidental. I'm not even sure how I managed to get a copy-paste wrong 😆